.gitignore

```diff
@@ -2,6 +2,7 @@
 *.o
 *.a
 *.so
+*.dylib
```

Comment:
Not sure if we have any *.h source code in future, but I think it's okay for now.